### DISCUSSION TITLE
Матрица 40х10 горит только 256 диодов.

### DISCUSSION THREAD

**Discussion by Crezidarkman** [vvip-68/GyverPanelWiFi#283] [Задавайте вопросы] [answered]
Пробовал разные платы ESP32 и ESP8266 Матрица горит не вся( Менял сегменты окон, думая что проблема в ленте. все тоже самое. подключил Wled  все нормально. Куда копать? На версии проекта 13 прошлую зиму все работало ок

**Comment by vvip-68** [comment, marked as answer]
Слишком мало данных и вообще не понятен вопрос. Судя потому, что он задан в ветке версии 13 -тогда не заю. Надо хоть вывод в лог при старте смотреть... Если под фразой "На версии проекта 13 прошлую зиму все работало ок" понимать, что сейчас используется не 13 (почему тогда в 13 ветке вопрос?), а в 14-й - тогда нужно смотреть указанную длину и размеры в настройках подключения ленты в Web-интерфейсе. Что тут и тут прописано?

![image](https://github.com/vvip-68/GyverPanelWiFi/assets/39353139/2f4e294b-8213-4340-89e0-63658674169f)

![image](https://github.com/vvip-68/GyverPanelWiFi/assets/39353139/22c13948-b6c1-4dae-979d-854e150f259d)

**Comment by Crezidarkman** [comment]
Прошу прощения заголовок темы не прочел( Да спасла вторая картинка с указанием количества диодов в матрице .... Я думал количество генерится автоматом при указания размера матрицы.

**Reply by vvip-68** [reply]
Автоматом. Если это первый запуск после прошивки на чистый МК. Если уже были какие-то настройки и они сохранены в EEPROM / Файловой системе - то берутся они.
